UI/window-basic-main.cpp

```diff
@@ -410,6 +410,29 @@ void OBSBasic::UpdateVolumeControlsDecayRate()
 	}
 }
 
+void OBSBasic::UpdateVolumeControlsPeakMeterType()
+{
+	uint32_t peakMeterTypeIdx = config_get_uint(basicConfig, "Audio",
+			"PeakMeterType");
+
+	enum obs_peak_meter_type peakMeterType;
+	switch (peakMeterTypeIdx) {
+	case 0:
```

Comment:
Consider combining `case 0` and `default`.  Though, if you do, take care not to accidentally introduce an implicit-fallthrough warning.